### PR TITLE
Feature/rlp 1084/topologies

### PR DESCRIPTION
**Pull request by mauricioirace**
This pr aims to improve topologies definitions
CHANGE LOG:
- Rename nodes in topologies to make them more generic
- Remove unnecessary downcasting to LuminoNode when retrieving the nodes.
- Basic topologies' nodes, now are referenced by number instead of name
- Remove bootstrapping test.

### CODE REVIEW

File: test/utils/index.ts [hunk 1]
```diff
@@ -13,12 +12,12 @@ export function sleep(ms: number) {
 
 export async function verifyEnv(env: LuminoTestEnvironment): Promise<void> {
```
Comment (mortelli) [inline test/utils/index.ts]: if the bootstrapping test is being removed, i don't see any reason to keep this function around

**Review by mortelli — COMMENTED** [review]
a welcome change, but keep in mind this will throw up a lot of conflicts once #9 is merged

File: test/utils/index.ts (hunk 1 above)
Comment (JONAF2103) [reply test/utils/index.ts]: Agree

**Review by JONAF2103 — APPROVED** [review]
LGTM, good work here

File: test/cases/payments/p2p.test.ts [hunk 2]
```diff
@@ -1,99 +1,96 @@
...
         Timeouts.TEST
     );
 
     test(
-        '2 tokens, target node',
+        '2 tokens, bob node',
         async () => {
-            // pay initator -> target
-            await nodes.initiator.client.sdk.makePayment({
+            // pay initator -> bob
```
Comment (mortelli) [inline test/cases/payments/p2p.test.ts]: replace `initator` with `alice` in this file

**Review by mortelli — APPROVED** [review]
good stuff :+1: 

one small correction down below, plus a question and a comment:

1. if we are not using the `basic.json` and `advanced.json` topologies, why don't we just go ahead and delete them?
2. we are not using the RIF token in tests yet. this is OK, as it is out of scope for this PR. let's remember we aren't (yet) testing this, although it's still a part of the topologies we're handling atm

afaic point 2 can be covered through [RLP-1085](https://jirainfuy.atlassian.net/browse/RLP-1085)

**Comment by mauricioirace** [comment]
> * if we are not using the `basic.json` and `advanced.json` topologies, why don't we just go ahead and delete them?

We could remove them, but I think they are good examples or templates

**Pull request merged**